packages/amplify-cli-core/src/hooks/skipHooks.ts

```diff
@@ -3,9 +3,9 @@ import { skipHooksFilePath } from './hooksConstants';
 
 export function skipHooks(): boolean {
```

Comment:
should we change this to `verifyScriptsAreAllowed(errorMessage: string): void`? Then this function can check and throw if disabled to prevent the caller from having to handle the result correctly

Reply:
if we do this we should create two functions:
1. `areScriptsAllowed` - so we can use to skip hooks - we can't fail on their presence (existing behavior)
2. `verifyScriptsAllowed` which we use in overrides and custom and that throws.

Reply:
Will follow up with this in separate PR as it will be a larger refactor